From 4cf90b8c07a4b050edc0dca32fbe48fdd484c38a Mon Sep 17 00:00:00 2001
From: Sowmiya Sree Elavalagan <quic_ssreeela@quicinc.com>
Date: Tue, 5 Nov 2024 15:45:43 +0530
Subject: [PATCH] Makefile: Enable CONFIG_QCOM_RPROC_DISABLE_MPD_SUPPORT

Enable QCOM_RPROC_DISABLE_MPD_SUPPORT Kconfig to
disable various userPDs in remoteproc in DTS

Change-Id: Icb6cab4276810d193120d5eecc07417def476d6f
Signed-off-by: Sowmiya Sree Elavalagan <quic_ssreeela@quicinc.com>
---
 package/kernel/mac80211/Makefile | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/package/kernel/mac80211/Makefile b/package/kernel/mac80211/Makefile
index 04bb29b46db0..5a6d4a1e6fbc 100644
--- a/package/kernel/mac80211/Makefile
+++ b/package/kernel/mac80211/Makefile
@@ -138,7 +138,9 @@ define KernelPackage/mac80211
 	+kmod-qca-nss-ppe-vp \
 	+kmod-qca-nss-ppe-ds
   KCONFIG:=\
-	CONFIG_AVERAGE=y
+	CONFIG_AVERAGE=y \
+	CONFIG_QCOM_RPROC_DISABLE_MPD_SUPPORT=y
+
   FILES:= $(PKG_BUILD_DIR)/net/mac80211/mac80211.ko
   ABI_VERSION:=$(PKG_VERSION)-$(PKG_RELEASE)
   MENU:=1
-- 
2.34.1

